backend/flask_app.py

```diff
@@ -0,0 +1,46 @@
+from flask import Flask, Response
```

Comment:
The current file name is not a problem, but the common approach to name it just `app.py`